src/datatypes/numeric.js

```diff
@@ -24,7 +24,14 @@ function generateFunctions(bufferReader,bufferWriter,size)
     };
   };
   var writer=function(value, buffer, offset) {
-    buffer[bufferWriter](value, offset);
+    try {
```

Comment:
Hmm there was something wrong with numbers due to something else I think. Yeah you're right that one should be removed.

I turned the return {error:} to a throw Error in write because of https://github.com/PrismarineJS/node-minecraft-protocol/blob/4752a0d759d3828bd358bad389c1f587feffbfa7/src/transforms/serializer.js#L92 , so I guess that should be fixed one way or the other.

Adding context through that error object you do it indeed.

The problem with the debugging function is it displays every packet, kind of hard to find the interesting part in the ouput when enabled, but yeah I see your point.